algorithms/linfa-kernel/Cargo.toml

```diff
@@ -24,8 +24,9 @@ default-features = false
 features = ["std", "derive"]
 
 [dependencies]
-ndarray = "0.13"
-sprs = { version = "0.9.3", default-features = false }
+ndarray = "0.14"
+num-traits = "0.2"
+sprs = { git="https://github.com/vbarrielle/sprs.git", rev="761d4f0", default-features = false }
```

Comment:
didn't know that you can specify the commit hash in your dependency tree :) to what revision does this point, I assume the `linfa-kernel` would not compile with the latest `sprs` release?

Reply:
Yes, `linfa-kernel` can not compile with `sprs 0.10`, my question was: could we release linfa 0.4 using such spec or is it too bold? Sure I would prefer a `sprs 0.10.1` fixing the issue but atm it blocks the ndarray upgrade.

Reply:
no that is not possible, because `crates.io` only allows self referencing and we would therefore not be able to publish the crate when using a foreign git source. We can't go forward without `sprs` fixing and publishing their crate.